frontend/index.html

```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
```

Comment:
<meta> tags are self closing tags in HTML5 and the trailing slash is optional. only XHTML or XML requires it.

Reply:
Thanks - I changed to follow this standard then. (A code reformatter would help here to keep a single shared style)